eksctl/shared-hubs-cluster.jsonnet

```diff
@@ -0,0 +1,112 @@
+// Exports an eksctl config file for carbonplan cluster
+local ng = import "./libsonnet/nodegroup.jsonnet";
+
+// place all cluster nodes here
+local clusterRegion = "us-west-2";
+local masterAzs = ["us-west-2a", "us-west-2b", "us-west-2c"];
+local nodeAz = "us-west-2a";
+
+// Node definitions for notebook nodes. Config here is merged
+// with our notebook node definition.
+// A `node.kubernetes.io/instance-type label is added, so pods
+// can request a particular kind of node with a nodeSelector
+local notebookNodes = [
+    { instanceType: "m5.large" },
+    { instanceType: "m5.xlarge" },
+    { instanceType: "m5.2xlarge" },
+    { instanceType: "m5.8xlarge" },
+];
+
+local daskNodes =
+    if "daskhub" == "daskhub" then [
+    // Node definitions for dask worker nodes. Config here is merged
+    // with our dask worker node definition, which uses spot instances.
+    // A `node.kubernetes.io/instance-type label is set to the name of the
+    // *first* item in instanceDistribution.instanceTypes, to match
+    // what we do with notebook nodes. Pods can request a particular
+    // kind of node with a nodeSelector
+        { instancesDistribution+: { instanceTypes: ["m5.large"] }},
+        { instancesDistribution+: { instanceTypes: ["m5.xlarge"] }},
+        { instancesDistribution+: { instanceTypes: ["m5.2xlarge"] }},
+        { instancesDistribution+: { instanceTypes: ["m5.8xlarge"] }},
+    ];
+{
+    apiVersion: 'eksctl.io/v1alpha5',
+    kind: 'ClusterConfig',
+    metadata+: {
+        name: "shared-hubs-cluster",
+        region: clusterRegion,
+        // Warning: version 1.23 introduces some breaking changes
+        // Checkout the docs before upgrading
+        // ref: https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi-migration-faq.html
+        version: '1.22'
```

Comment:
Explicitly stating that this is ok for now to get this out the door, and we should work on upgrading in the new year.

Reply:
Yes! Scale to zero node groups!